packages/react-aria-components/src/Table.tsx

```diff
@@ -1329,6 +1343,7 @@ export const Cell = /*#__PURE__*/ createLeafComponent(TableCellNode, (props: Cel
     }
   });
 
+  // TODO: Lint doesn't catch these, it thinks we're not in a component render cycle here?
```

Comment:
I'm not sure what to make of this yet, but not going to handle it in this series of PRs

Reply:
ok, went ahead and updated them

I'm mostly worried that the compiler isn't looking through these components, so won't do any optimisations
I'll raise an issue with React at some point about it probably